kitchen.yml

```diff
@@ -50,6 +50,18 @@ suites:
     verifier:
       inspec_tests:
         - path: test/integration/server_install
+  - name: server_install_10.11
+    run_list:
+      - recipe[test::server_install]
+    attributes:
+      mariadb_server_test_version: '10.11'
+    verifier:
+      inspec_tests:
+        - path: test/integration/server_install
+    excludes:
+      # 10.11 is too bleeding edge for these platforms
+      - ubuntu-18.04
+      - centos-7
```

Comment:
Please update `.github/workflows/ci.yml` so that this new suite gets run in CI. Also make sure you exclude these two suites.

Reply:
Done in bb4f1724baad27d4dc531f082f44144782b9c14d